recipes/apr/all/patches/0007-cross-compile-fix-gen_test_char.patch

```diff
@@ -0,0 +1,12 @@
+--- Makefile.in
++++ Makefile.in
+@@ -134,7 +134,7 @@ 
+ OBJECTS_gen_test_char = tools/gen_test_char.lo $(LOCAL_LIBS)
+ tools/gen_test_char.lo: tools/gen_test_char.c
+ 	$(APR_MKDIR) tools
+-	$(LT_COMPILE)
++	gcc -c $< -o $@

```

Comment:
This creates a dependency on `gcc`. Users might want to use a system clang compiler.
Perhaps use `$(BUILD_CC)` instead?
```suggestion
+	$(BUILD_CC) -c $< -o $@

```

Reply:
This did not work, it still used the cross compiler rather than the local machine one